install_config/configuring_gce.adoc

```diff
@@ -75,4 +134,60 @@ container. Therefore, *_node-config.yaml_* should be in *_/etc/origin/node_*
 instead of *_/etc/_*.
 ====
 
-Then, start or restart the {product-title} services on the master and all nodes.
+. Start or restart the {product-title} services all nodes.
++
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-node
+----
+endif::[]
+
+
+[[configuring-gce-multizone]]
+
+== Configuring Multizone Support in a GCE Deployment
+
+// From https://bugzilla.redhat.com/show_bug.cgi?id=1390160
```

Comment:
We tend to avoid commenting directly in the docs, so I recommend editing this BZ link out and just leaving a paper trail in the PR description/comments if necessary.